src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -720,5 +740,10 @@ private static HashSet<string> GetMatchingFrameworkStrings(PackageSpec spec, NuG
 
             return result;
         }
+
+        private static XElement GeneratePackagePathProperty(string packageName, string packagePath)
+        {
+            return GenerateProperty($"Pkg{packageName.Replace(".", "_")}", packagePath);
```

Comment:
Where are we adding condition for `Pkg<package_path>` already exists or not on this property?

Reply:
They are added by `GenerateProperty` on line `191`